packages/web/package.json

```
@@ -9,16 +9,16 @@
   "types": "src/index.d.ts",
   "license": "MIT",
   "dependencies": {
-    "@apollo/react-components": "^3.1.3",
-    "@apollo/react-hooks": "^3.1.3",
+    "@apollo/client": "^3.1.3",
+    "@apollo/react-components": "^4.0.0",
+    "@apollo/react-hooks": "^4.0.0",
     "@redwoodjs/forms": "^0.16.0",
     "@types/react": "16.9.38",
-    "apollo-boost": "0.4.7",
-    "apollo-cache": "^1.3.4",
-    "apollo-client": "^2.6.8",
-    "apollo-link": "^1.2.13",
-    "apollo-utilities": "^1.3.3",
-    "core-js": "3.6.4",
+    "apollo-boost": "0.4.9",
+    "apollo-cache": "^1.3.5",
+    "apollo-link": "^1.2.14",
+    "apollo-utilities": "^1.3.4",
+    "core-js": "3.6.5",
```

`core-js` should stay the same across all of our packages. I should add the resolutions stuff to the main package to pin it, maybe we can upgrade in a seperate PR and I can show you why.

```suggestion
    "core-js": "3.6.4",
```

---

Ah, roger that. Understood and didn't intend for this to be included.